internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -533,3 +534,12 @@ type Ownership struct {
 	// TODO: Polymorphic associations won't enforce foreign key to entities in different tables
 	// we might need to have separated ownership tables for each entity.
 }
+
+type Value struct {
+	EnvID sdktypes.UUID `gorm:"primaryKey;type:uuid;not null"`
```

Comment:
Let's link it to the project directly instead of env

I think we should refactor to remove envs as a whole and adding more stuff to it right now won't help

What do you think?

Reply:
i think we should keep env for now. we'll probably kill them, but i want it to be somewhat consistent.

Reply:
ok, i will change to project.

Reply:
done